source/_posts/2020-09-18-mac-companion.markdown

```diff
@@ -0,0 +1,57 @@
+
+<p class='img'>
+<img src='/images/blog/2020-09-18-mac-companion/render.png' alt='Rendered image of the Home Assistant Lovelace interface running full screen on a 16-inch Mac Book Pro'></a>
+Rendered image of the Home Assistant Lovelace interface running full screen on a 16-inch Mac Book Pro.
+</p>
+
+## Automate when you start using your webcam or microphone
+
+Let's start by looking at some of the coolest Mac-specific features of the app, sensors for all your webcams and microphones. Each camera and microphone has its own `binary_sensor` showing whether it is active or not. These can enable some really useful automations, especially for those home working at the moment. You could automatically turn off the radio when answering a call or close the blinds behind you to improve your video quality. To see just how useful this can be in the real world, check out this video of how our very own Frenck is using these sensors in his streaming set up.
+
+TODO: ADD VIDEO EMBED HERE
+
+One huge advantage of running on a Mac compared to a mobile device is the much larger battery. This means we are not constrained by battery saving measures and can address one of the most common gripes with the iOS app, update intervals. On a Mac entity updates are triggered immediately when something changes. You will see this reported by the `sensor.DEVICE_NAME_last_update_trigger` reporting `Signaled`.
+
+## Notifications
+
+Just like the iOS app, you can send notifications to your Mac with services like `notify.mobile_app_DEVICE_NAME`. One small difference is that [critical notifications](https://companion.home-assistant.io/docs/notifications/critical-notifications) are not yet available for the Mac app. However, all our other notifications features like actionable notifications work on the Mac app. To see what is possible, take a look at [the docs](https://companion.home-assistant.io/).
+
+## Interface
+
+Your Mac isn't a mobile so you shouldn't be forced to use an interface designed for a mobile. In the Mac app the App Configuration page has been removed from Home Assistant's sidebar and instead the configuration options and preferences are in the Home Assistant menu on the menu bar right where you'd expect to find them for any other app and all the standard shortcuts work too (just hit `cmd + ,` for preferences). Here you will also find an option to manually send an update to Home Assistant and you can even open extra Home Assistant windows! Finally, on the menu bar you can also find a new Actions menu where you can see all your [actions](https://companion.home-assistant.io/docs/core/actions) and fire them.
```

Comment:
```suggestion
The Mac App is definitely a _Mac_ app. With support for multiple windows, you'll find the usual shortcuts (like Preferences under `⌘,` instead of living in the sidebar) as well as the ability to create multiple Lovelace windows under File > New. 

In the menu bar you will also find an option to manually send an update to Home Assistant and a new Actions menu where you can see all your [actions](https://companion.home-assistant.io/docs/core/actions) and fire them.
```

or something like this

Reply:
Think I've got something close to this now